### ISSUE TITLE
Google Maps Edge Case

### ISSUE THREAD

**Issue by Kdog0073** [PokeNavBot/issue-tracker#340]
Currently, pokenav uses a directions link that looks similar to this: https://www.google.com/maps/dir/?api=1&destination=42.04144%2C+-87.909073

What seems to happen is this URL converts the location into an address. The problem is that this address can sometimes omit an "implied" directional (E, W, N, S, etc.) when the street name on the map itself does not have one. This has led to some users being sent to a completely wrong location.

Other discord bots in the area using google tend to use this type of url:
https://www.google.com/maps?q=42.010954,-88.17276

This seems to work as the latitude and longitude is not converted into a physical address. Users typically need to press some sort of "directions to" button, but users have not found this to be problematic.

**Comment by dandesousa** [comment]
@Kdog0073 Is the above coordinate producing the issue and messing people up depending on where they are coming from? While I recognize there appears to be reverse geocoding going on -- does it actually produce the wrong address? I don't believe that the query others are using is "correct" from an API perspective, but I would like to construct one that avoids reverse geocoding.

**Comment by Kdog0073** [comment]
The above coordinates were just example URLs that I grabbed from the bots. I included an example situation below-

It is an odd situation since most cities do include the direction label on the addresses. It also does not seem consistent (different results for different people). For example, this happens at Gym ID 238174
![image](https://user-images.githubusercontent.com/6502797/54854487-0c48af80-4cc1-11e9-97c7-1e6782e771a6.png)
